plus/provider/src/main/webapp/static/waterfall-light.js

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
This file should not be included here, as it is a js lib

Reply:
Agreed - Moved